mock/src/main/java/com/tngtech/keycloakmock/api/TokenConfig.java

```diff
@@ -305,6 +332,22 @@ public Builder withSourceToken(@Nonnull final String originalToken) {
               clazz, entry.getKey(), entry.getValue().getClass()));
     }
 
+    @Nonnull
+    private String getRealm(@Nonnull final URI issuer) {
+      String[] path = issuer.getPath().split("/");
+      if (path.length != 4
```

Comment:
Could this not be checked in a more expressive manner by matching against a regex?

Reply:
Yes, good point.